front/app/modules/commercial/flag_inappropriate_content/admin/components/InappropriateContentWarning.tsx

```diff
@@ -1,6 +1,13 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Icon, colors } from 'cl2-component-library';
+import { isNilOrError } from 'utils/helperUtils';
+
+// services
+import { getFlagType } from '../../services/inappropriateContentFlags';
```

Comment:
Not sure if this function should be in the services

Reply:
I agree! There is a convention to create `utils` files for the modules, maybe it should go there?

Reply:
Moved it to a utils file inside this module.